2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Configure stats pid to 19648
2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Loading settings from C:\Users\sevitha\.config\wandb\settings
2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Loading settings from F:\IIT_final_yr\Myproject\FypModel\pythonProject8\wandb\settings
2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-26 19:26:19,051 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': 'F:\\IIT_final_yr\\Myproject\\FypModel\\pythonProject8\\main.py', 'program': 'F:\\IIT_final_yr\\Myproject\\FypModel\\pythonProject8\\main.py'}
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:_log_setup():526] Logging user logs to F:\IIT_final_yr\Myproject\FypModel\pythonProject8\wandb\run-20240126_192618-m2vcign2\logs\debug.log
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:_log_setup():527] Logging internal logs to F:\IIT_final_yr\Myproject\FypModel\pythonProject8\wandb\run-20240126_192618-m2vcign2\logs\debug-internal.log
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:init():566] calling init triggers
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'SYSTEM': CfgNode({'NUM_GPU': 1, 'NUM_WORKERS': 4}), 'WANDB': CfgNode({'PROJECT_NAME': 'vcnet-blind-image-inpainting', 'ENTITY': 'vvgl-ozu', 'RUN': 16, 'LOG_DIR': './logs/VCNet_Raindrop_302000step_4bs_0.0002lr_1gpu_16run', 'NUM_ROW': 0}), 'TRAIN': CfgNode({'NUM_TOTAL_STEP': 302000, 'START_STEP': 0, 'BATCH_SIZE': 4, 'SHUFFLE': True, 'NUM_STEPS_FOR_JOINT': 20000, 'LOG_INTERVAL': 200, 'SAVE_INTERVAL': 10000, 'SAVE_DIR': './weights', 'RESUME': True, 'VISUALIZE_INTERVAL': 200, 'TUNE': False}), 'MODEL': CfgNode({'NAME': 'VCNet', 'IS_TRAIN': True, 'RAINDROP_TUNE': True, 'RAINDROP_WEIGHTS': None, 'RAINDROP_VISUALIZE_INTERVAL': 20, 'RAINDROP_LOG_INTERVAL': 20, 'RAINDROP_SAVE_INTERVAL': 200, 'MPN': CfgNode({'NAME': 'MaskPredictionNetwork', 'NUM_CHANNELS': 64, 'NECK_CHANNELS': 128, 'LR': 0.001, 'BETAS': (0.5, 0.9), 'SCHEDULER': [], 'DECAY_RATE': 0.0, 'LOSS_COEFF': 2.0}), 'RIN': CfgNode({'NAME': 'RobustInpaintingNetwork', 'NUM_CHANNELS': 32, 'NECK_CHANNELS': 128, 'LR': 0.0001, 'BETAS': (0.5, 0.9), 'SCHEDULER': [], 'DECAY_RATE': 0.0, 'LOSS_COEFF': 1.0, 'EMBRACE': True}), 'D': CfgNode({'NAME': '1-ChOutputDiscriminator', 'NUM_CHANNELS': 64, 'LR': 0.001, 'BETAS': (0.5, 0.9), 'SCHEDULER': [], 'DECAY_RATE': 0.5, 'NUM_CRITICS': 5}), 'JOINT': CfgNode({'NAME': 'JointNetwork', 'LR': 0.0002, 'BETAS': (0.5, 0.9), 'SCHEDULER': [], 'DECAY_RATE': 0.5})}), 'OPTIM': CfgNode({'GP': 10, 'MASK': 1, 'RECON': 1.4, 'SEMANTIC': 0.0001, 'TEXTURE': 0.001, 'ADVERSARIAL': 0.001}), 'DATASET': CfgNode({'NAME': 'Raindrop', 'ROOT': './datasets/ffhq/images1024x1024', 'CONT_ROOT': './datasets/CelebAMask-HQ', 'IMAGENET': './datasets/ImageNet/', 'RAINDROP_ROOT': './datasets/raindrop/train20/train', 'SIZE': 256, 'MEAN': [0.5, 0.5, 0.5], 'STD': [0.5, 0.5, 0.5]}), 'MASK': CfgNode({'MIN_NUM_VERTEX': 8, 'MAX_NUM_VERTEX': 24, 'MEAN_ANGLE': 1.2566370614359172, 'ANGLE_RANGE': 0.20943951023931953, 'MIN_WIDTH': 8, 'MAX_WIDTH': 16, 'NUM_ITER_SMOOTHING': 3, 'MIN_REMOVAL_RATIO': 0.2, 'MAX_REMOVAL_RATIO': 0.5, 'GAUS_K_SIZE': 15, 'SIGMA': 4}), 'TEST': CfgNode({'OUTPUT_DIR': './outputs', 'ABLATION': False, 'WEIGHTS': None, 'BATCH_SIZE': 64, 'ITER': 4, 'MODE': 7, 'IMG_ID': 52, 'C_IMG_ID': 38, 'BRUSH_COLOR': 'RED', 'GRAFFITI_PATH': './datasets/graffiti-dataset/dataset/graffiti_sample/000001ff0013ffff.p', 'TEXT': 'hello world', 'FONT': '/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf', 'FONT_SIZE': 24})}
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:init():616] starting backend
2024-01-26 19:26:19,052 INFO    MainThread:19648 [wandb_init.py:init():620] setting up manager
2024-01-26 19:26:19,058 INFO    MainThread:19648 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-01-26 19:26:19,068 INFO    MainThread:19648 [wandb_init.py:init():628] backend started and connected
2024-01-26 19:26:19,077 INFO    MainThread:19648 [wandb_init.py:init():720] updated telemetry
2024-01-26 19:26:19,155 INFO    MainThread:19648 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-26 19:26:27,776 ERROR   MainThread:19648 [wandb_init.py:init():779] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2024-01-26 19:26:29,804 ERROR   MainThread:19648 [wandb_init.py:init():1194] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "C:\Users\sevitha\.conda\envs\pythonProject8\Lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
          ^^^^^^^^^
  File "C:\Users\sevitha\.conda\envs\pythonProject8\Lib\site-packages\wandb\sdk\wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2024-01-26 19:26:32,039 WARNING MsgRouterThr:19648 [router.py:message_loop():77] message_loop has been closed
